modules/1_prepare/prepare.tf

```diff
@@ -153,6 +153,18 @@ EOF
         ]
     }
 
+    provisioner "remote-exec" {
+        inline = [<<EOF
+if [ '${var.rhel_subscription_username}' != '' ] || [ '${var.rhel_subscription_org}' != '' ]
+then
+    sudo subscription-manager repos --enable ${var.ansible_repo_name}
+else
+    sudo yum install -y epel-release
```

Comment:
Since distro version is 8+, we should follow the instructions here for EPEL - https://fedoraproject.org/wiki/EPEL